sidebarBuild.ts

```diff
@@ -0,0 +1,209 @@
+import {SidebarsConfig} from '@docusaurus/plugin-content-docs';
+
+export const sidebarBuild: SidebarsConfig = {
+  developers: [
+    {
+      type: 'doc',
+      label: 'Overview',
+      id: 'build/README',
+    },
+    {
+      type: 'category',
+      label: 'Sapphire',
+      link: {
```

Comment:
I think it's room for this:

```suggestion
      collapsible: false,
      link: {
```

And you can drop `collapsed: false` lines.